NitroxServer/GameLogic/Spawning/EntitySpawner.cs

```diff
@@ -106,5 +110,74 @@ private void SpawnEntities()
                 }
             }
         }
+
+        private bool GetDataFiles(out string lootDistributions, out Dictionary<string, WorldEntityInfo> worldEntityData)
```

Comment:
Shouldn't this be `static`, because you're using `out` to write into a member field and are not accessing anything else?

Reply:
Good point. The template was copied from the library itself, and it uses some weird practices since it was ported from c++.